sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -154,6 +175,20 @@ def __init__(self, locations: List[sbol3.Location] = [], **kwargs) -> None:
             self.qualifier_value    = sbol3.TextProperty(self, f"{self.GENBANK_FEATURE_QUALIFIER_NS}#value", 0, math.inf)
 
 
+    class Range_GenBank_Extension(sbol3.Range):
```

Comment:
I'm thinking this should be a child of sbol3.Location instead of sbol3.Range, because something that treats an inexact range as an exact range will have errors.

Reply:
With the new workaround merged for the pySBOL3 merge, this seems to have been appropriately implemented now.

Reply:
@jakebeal I'm leaving it to you to resolve this requested change. I agree with @mohitdmak that the change we collaborated on yesterday resolves this issue.